.env.example

```
@@ -0,0 +1 @@
+JSEARECT=��i��hq�a��w�MPATRAPP_PI-APP_ID=UAQIAA}A$�AA}-e}%=L������輽�����̹չ����͠�����Ʌ������������
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)